### DISCUSSION TITLE
Easy way to find if and its corresponding end if with modern fortran ?

### DISCUSSION THREAD

**Discussion by Atria316** [krvajal/vscode-fortran-support#1065] [Q&A - deprecated see Fortran Discourse]
I've started using the extension in VS Code, and I'm using a very large, old script, which I'm not familiar with. I'd like to know if there's any simple way to identify if-end if /do-end do structures that I'm not aware of.  
The code is poorly indented, so collapsing the code doesn't help. 

I'm surprised I haven't found this question yet. I've done my research for similar questions before, maybe i'm not using the proper terminology ?
Thanks in advance !  

Edit : I'm new to Modern Fortran , to VS Code and Fortran in general. I've read the documentation before asking my question, too. I don't know if I'm missing something obvious, as in general, the extension doesn't work as I expected it to. If I right-click a `Call some_subroutine`, I should be redirected to the module containing that subroutine, right ?

**Comment by gnikit** [comment]
You can probably format your code. findent, is the default formatter. That should fix the indentation and you should be able to collapse then.
